x-pack/plugins/security_solution/public/common/components/event_details/overview/index.tsx

```diff
@@ -0,0 +1,178 @@
+  ALERTS_HEADERS_RULE,
+  ALERTS_HEADERS_SEVERITY,
+  SIGNAL_STATUS,
+} from '../../../../detections/components/alerts_table/translations';
+import {
+  SIGNAL_RULE_NAME_FIELD_NAME,
+  SIGNAL_STATUS_FIELD_NAME,
+} from '../../../../timelines/components/timeline/body/renderers/constants';
+import { FormattedFieldValue } from '../../../../timelines/components/timeline/body/renderers/formatted_field';
+import { OverviewCard, OverviewCardWithActions } from '../overview/overview_card';
+import { StatusPopoverButton } from '../overview/status_popover_button';
+import { SeverityBadge } from '../../../../../public/detections/components/rules/severity_badge';
+
+export const NotGrowingFlexGroup = euiStyled(EuiFlexGroup)`
+  flex-grow: 0;
+`;
+
+interface Props {
+  browserFields: BrowserFields;
+  contextId: string;
+  data: TimelineEventsDetailsItem[];
+  eventId: string;
+  handleOnEventClosed: () => void;
+  indexName: string;
+  timelineId: string;
+}
+
+export const Overview = React.memo<Props>(
+  ({ browserFields, contextId, data, eventId, handleOnEventClosed, indexName, timelineId }) => {
+    const statusData = useMemo(() => {
+      const item = find({ field: SIGNAL_STATUS_FIELD_NAME, category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const severityData = useMemo(() => {
+      const item = find({ field: 'kibana.alert.rule.severity', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const riskScoreData = useMemo(() => {
+      const item = find({ field: 'kibana.alert.rule.risk_score', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const ruleNameData = useMemo(() => {
+      const item = find({ field: SIGNAL_RULE_NAME_FIELD_NAME, category: 'kibana' }, data);
+      const linkValueField = find({ field: 'kibana.alert.rule.uuid', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+          linkValueField,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    return (
+      <NotGrowingFlexGroup gutterSize="s">
+        {hasData(statusData) && (
+          <EuiFlexItem>
+            <OverviewCard title={SIGNAL_STATUS}>
```

Comment:
consider using a technique similar to the other cards below to display hover actions for `Status`, as illustrated in the mockup below:

![status_mock](https://user-images.githubusercontent.com/4459398/144660230-dea87541-d06a-4beb-801f-ba5a3ffca60a.png)

Reply:
Woops, how did I forget that? I guess I paid too much attention to the video, rather than the Figma :D 
Thanks for point this out 👏  I added this in https://github.com/elastic/kibana/pull/120347/commits/8a7110a2cd7876c93ff16d82a2d7d25ba04c23e8